app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,8 @@
         <activity
             android:name=".ui.splash.SplashScreenActivity"
             android:noHistory="true"
+            android:screenOrientation="portrait"
+            android:configChanges="orientation|screenSize"
```

Comment:
Dlaczego dodałeś też `android:configChanges="orientation|screenSize"`?

Reply:
https://developer.android.com/guide/topics/resources/runtime-changes#HandlingTheChange
Według przewodnika nie dodanie tego, spowoduje, że aktywność i tak zostanie odtworzona, ale w tej samej orientacji.

Reply:
Ale w tym zadaniu  chyba chodzi tylko o zablokowanie orientacji do portretu, a nie o zapobiegnięcie zmiany configu? Chyba  chodzi tylko o to, że UI jest przygotowany tylko pod portret?

Reply:
Przetestowałem i okazuje się, że `configChanges` było zbędne, także wywaliłem. Dzięki. :)